libs/martech/martech.js

```diff
@@ -95,20 +95,19 @@ const getDtmLib = (env) => ({
       : env.consumer?.marTechUrl || 'https://assets.adobedtm.com/d4d114c60e50/a0e989131fd5/launch-2c94beadc94f-development.min.js',
 });
 
-export default async function init({ persEnabled = false, persManifests, utils: ogUtils }) {
-  utils = ogUtils;
-  const config = utils.getConfig();
+export default async function init({ persEnabled = false, persManifests }) {
+  const config = getConfig();
 
   const { url, edgeConfigId } = getDtmLib(config.env);
-  utils.loadLink(url, { as: 'script', rel: 'preload' });
+  loadLink(url, { as: 'script', rel: 'preload' });
 
   if (persEnabled) {
-    utils.loadLink(
+    loadLink(
       `${config.miloLibs || config.codeRoot}/features/personalization/personalization.js`,
       { as: 'script', rel: 'modulepreload' },
     );
-    utils.loadLink(
-      `${config.miloLibs || config.codeRoot}/features/personalization/manifest-utils.js`,
+    loadLink(
+      `${config.miloLibs || config.codeRoot}/features/personalization/manifest-js`,
```

Comment:
Typo? Or did you mean to write `manifest-js`?

Reply:
doh thanks!